hadoop-ozone/dist/src/main/compose/ozone-csi/docker-compose.yaml

```diff
@@ -23,6 +23,8 @@ services:
       - ../..:/opt/hadoop
     env_file:
       - docker-config
+    environment:
+      HADOOP_OPTS: ${HADOOP_OPTS}
```

Comment:
I think we can avoid these by passing the variable via `-e` option of `docker-compose`, similar to how it is done for some other variables (`SECURITY_ENABLED`, etc.):

https://github.com/apache/hadoop-ozone/blob/53395a0ed75d96575a7dc26a7adec4eefabb2b74/hadoop-ozone/dist/src/main/compose/testlib.sh#L107

Reply:
We need it for `docker-compose up`. Do you see any possibility to set generic environment variables for all the containers? This approach works only for `docker-compose exec` IMHO.

Reply:
Sorry, I confused the two commands.